web/src/lib/client/users.js

```diff
@@ -49,6 +49,16 @@ export default class UsersClient {
     return proxy.RootPasswordSet;
   }
 
+  /**
+   * Return string with ssh key or empty string
+   *
+   * @return {Promise.<String>}
+   */
+  async rootSSHKey() {
```

Comment:
We are using `get*` for that kind of functions. We can change it, for sure, but we should be consistent. So I would use getRootSSHKey (and I am still not sure about capitalization).